go/ct/rlz/conditions.go

```diff
@@ -1053,4 +1053,70 @@ func (c *hasNoBlobHash) String() string {
 	return fmt.Sprintf("%v does not have BlobHash", c.index.String())
 }
 
-////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+// EIP 7702
```

Comment:
I think leaving this comment does not help much. I would rather see `Delegation Designation - https://eips.ethereum.org/EIPS/eip-7702`. If nothing else for consistency